bin/check.sh

```diff
@@ -11,7 +13,9 @@ gometalinter --concurrency=4 --enable-gc --deadline=300s --disable-all\
   --enable=goconst\
   --enable=gofmt\
   --enable=goimports\
-  --enable=golint --exclude=.pb.go\
+  --enable=golint\
+  --exclude=.pb.go\
+  --exclude=_test.go\
```

Comment:
Can we name them as "*_test.gen.go" or something like that? We also need these as exclusions to code coverage.